config/sync/migrate_plus.migration.solo_opportunities.yml

```
@@ -0,0 +1,287 @@
+uuid: 18fe55a6-a1f6-47c9-b079-992ef831c99b
+langcode: en
+status: true
+dependencies: {  }
+id: solo_opportunities
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags: {  }
+migration_group: opportunities
+label: 'SOLO Opportunities'
+source:
+  track_changes: true
+  plugin: url
+  data_fetcher_plugin: file
+  data_parser_plugin: simple_xml
+  urls:
+    - 'https://solo.stanford.edu/data/opportunities.xml?type=236'
+  constants:
+    status: 1
+    type: su_opportunity
+    basic_html: stanford_html
+    image_destination: 'public://media/image/solo/'
+    apply_link_title: 'Apply'
+    learn_more_link_title: 'Learn More'
+  item_selector: /OPPORTUNITIES/OPPORTUNITY
+  fields:
+    -
+      name: id
+      label: ID
+      selector: ID
+    -
+      name: title
+      label: Title
+      selector: TITLE
+    -
+      name: url
+      label: URL
+      selector: URL
+    -
+      name: submission_open
+      label: 'Submission Open Date'
+      selector: SUBMISSIONS_OPEN
+    -
+      name: application_deadline
+      label: 'Application Deadline'
+      selector: APPLICATION_DEADLINE
+    -
+      name: start
+      label: Start
+      selector: DURATION/START
+    -
+      name: end
+      label: End
+      selector: DURATION/END
+    -
+      name: sponsor
+      label: Sponsor
+      selector: SPONSOR
+    -
+      name: teaser
+      label: Teaser
+      selector: TEASER
+    -
+      name: body
+      label: Body
+      selector: DESCRIPTION
+    -
+      name: image
+      label: Image
+      selector: IMAGE
+    -
+      name: eligibility
+      label: Eligibility
+      selector: ELIGIBILITY
+    -
+      name: type
+      label: Type
+      selector: TYPE
+    -
+      name: time_of_year
+      label: 'Time of the Year'
+      selector: TIME_OF_YEAR
+    -
+      name: open_to
+      label: 'Open To'
+      selector: OPEN_TO
+    -
+      name: location
+      label: Location
+      selector: LOCATION
+    -
+      name: contact_name
+      label: 'Contact Name'
+      selector: CONTACT_NAME
+    -
+      name: contact_email
+      label: 'Contact Email'
+      selector: CONTACT_EMAIL
+    -
+      name: contact_phone
+      label: 'Contact Phone Number'
+      selector: CONTACT_PHONE
+    -
+      name: learn_more
+      label: 'Learn More Link'
+      selector: LEARN_MORE
+  ids:
+    id:
+      type: string
+process:
+  title: title
+  revision_translation_affected:
+    -
+      plugin: default_value
+      default_value: 1
+  revision_timestamp:
+    plugin: callback
+    callable: cardinal_service_profile_get_time
```

```suggestion
    callable: time
```

😂 

_THAT'S A JOKE, SON!_

---

yeah make that change, then run `drush @cardinalservice.local cim -y; drush @cardinalservice.local mim solo_opportunities --update` you'll see the messages

---

I'll believe you on that; don't need to do it myself 😄